### DISCUSSION TITLE
Add broadcasts of achievements to friends

### DISCUSSION THREAD

**Discussion by vaqah** [runelite/runelite#15043] [Ideas]
Having a option to get in-game messages in chat of achievements or high value drops from ~~clan chat members and~~ friends.

So XXquickscopes has just received a bandos chestplate. 
     Introverted spud has just achieved 99 in firemaking.

**Comment by Stickaplex** [comment]
Curious...if this was implemented would it be for any1 or only those who have runelite. Obv pref the first one but curious on how it would work from a coding standpoint

**Comment by abextm** [comment]
This would have the same problem as !kc would. see #306.
If we didn't care about fake data, we could make it work as long as any RuneLite user could see the event happening.

**Comment by Enriath** [comment]
This exists for clan chats with Jagex's clan implementation
Editing the issue to be about friends, since that's not something that can be done ingame
